include/geom/elem.h

```diff
@@ -2280,6 +2290,32 @@ Elem::build_side_ptr (std::unique_ptr<const Elem> & elem,
 
 
 
+template <typename Sideclass, typename Subclass>
+inline
+std::unique_ptr<Elem>
+Elem::simple_build_side_ptr (const unsigned int i,
```

Comment:
This doesn't set the subdomain id? The functions you are replacing all set the subdomain id of the side... maybe something is relying on this behavior?

Reply:
Ah, I'm guessing sides now get their subdomain_id from their interior_parent... if that's the case, nevermind!

Reply:
They get their subdomain_id via the parent argument to their constructor, which we later turn into their interior_parent, yeah.  I admit I'd originally forgotten about that behavior too - I was all ready to fix a "bug" in the optimized build_side_ptr overload where it was explicitly resetting subdomain_id when modifying a side_ptr but seemed to be failing to do so when first constructing one.